ksa_compliance/templates/simplified_e_invoice.xml

```diff
@@ -220,7 +220,7 @@
             </cac:ClassifiedTaxCategory>
         </cac:Item>
         <cac:Price>
-            <cbc:PriceAmount currencyID="{{ invoice.currency_code }}">{{ item.net_amount or item.amount | round(2) }}</cbc:PriceAmount>
+            <cbc:PriceAmount currencyID="{{ invoice.currency_code }}">{{ item.net_amount | round(2) or item.amount | round(2) }}</cbc:PriceAmount>
```

Comment:
Instead of repeating the rounding, I think we can surround the `or` expression with brackets, e.g. `(item.net_amount or item.amount) | round(2)`